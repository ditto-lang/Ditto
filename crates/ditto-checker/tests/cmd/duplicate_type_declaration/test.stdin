module Test exports (..)

type A = A




type A = B | C
