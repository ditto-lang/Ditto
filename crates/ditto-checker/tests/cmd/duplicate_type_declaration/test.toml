bin.name = "ditto-checker-testbin-check-module"
args = ["duplicate_type_declaration.ditto"]
fs.sandbox = true
status = "failed"
