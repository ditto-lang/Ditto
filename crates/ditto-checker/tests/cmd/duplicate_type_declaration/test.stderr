
  × duplicate type declaration
   ╭─[duplicate_type_declaration.ditto:2:1]
 2 │ 
 3 │ type A = A
   ·      ┬
   ·      ╰── previously defined here
 4 │ 
   ╰────
   ╭─[duplicate_type_declaration.ditto:7:1]
 7 │ 
 8 │ type A = B | C
   ·      ┬
   ·      ╰── can't be redefined here
   ╰────

